interface/super/edit_list.php

```diff
@@ -1023,6 +1045,14 @@ function mysubmit() {
                     <select name='list_id' class="form-control select-dropdown"
                             id="list_id">
                         <?php
+                        /*
+                         * Keep proper list name (otherwise list name changes according to
+                         * the options shown on the screen).
+                         */
+                        $list_id_container = trim(strip_tags($_GET["list_id_container"]));
```

Comment:
hi @yurige-matrix , What is the strip_tags() call meant to prevent?

Reply:
hi, @bradymiller , prevent html in url,